examples/batch-cp2k/reference-trajectory.py

```diff
@@ -136,17 +148,25 @@ def write_cp2k_in(
 
 
 # %%
+#
+# We will now download basis set files from CP2K website. Depending on your CP2K
+# installation, this might not be necessary!
 
 
-def mkdir_force(*args, **kwargs) -> None:
-    """Warpper to ``os.mkdir``.
+def download_parameter(file):
+    path = os.path.join("parameters", file)
+
+    if not os.path.exists(path):
+        url = f"https://raw.githubusercontent.com/cp2k/cp2k/master/data/{file}"
```

Comment:
Should we maybe not use the master but rather a fixed tag? I don't expect that they change how basis sets are stored but just to be sure.

```suggestion
        url = f"https://raw.githubusercontent.com/cp2k/cp2k/support/v2024.1/data/{file}"
```